src/tokens/theme.native.js

```diff
@@ -0,0 +1,29 @@
+import { fonts, colors } from '../tokens';
+import { lightTheme as baseLightTheme, darkTheme as baseDarkTheme } from './theme.web';
+
+// For  styled components in React Native: converts unitless numbers to "px" strings
+const getPxValue = (value) => `${value}px`;
+
+const convertToPx = (property) =>
+  Object.keys(property).reduce((acc, val) => ({ ...acc, [val]: getPxValue(property[val]) }), {});
+
+const nativeFonts = {
+  ...fonts,
+  lineHeight: convertToPx(fonts.lineHeight),
+  letterSpacing: convertToPx(fonts.letterSpacing),
+};
+
+const lightTheme = {
+  ...baseLightTheme,
+  fonts: nativeFonts,
+};
+
+const darkTheme = {
+  ...baseDarkTheme,
+  fonts: nativeFonts,
+};
+
+const theme = { ...lightTheme, ...colors };
```

Comment:
Why are you adding all the colors into theme?

Reply:
colors goes into default `theme` export and not in `lightTheme` and `darkTheme`.

Reply:
I thought we concluded that default theme == light theme

Reply:
@shubhnik We need to just say `const theme = lightTheme`. That's what we all concluded yesterday